flask/app.py

```diff
@@ -1370,6 +1390,33 @@ def url_defaults(self, f):
         self.url_default_functions.setdefault(None, []).append(f)
         return f
 
+    def _find_error_handler(self, e):
+        """Finds a registered error handler for the request’s blueprint.
+        If neither blueprint nor App has a suitable handler registered, returns None
+        """
+        exc_class, code = self._get_exc_class_and_code(type(e))
+        
+        def find_superclass(d):
+            if not d:
+                return None
+            for superclass in exc_class.mro():
```

Comment:
From [SO](https://stackoverflow.com/questions/19713585/whats-the-difference-between-the-mro-method-and-the-mro-attribute-of-a-clas):

>mro() is called on instanciation and stores its result in __mro__. They don't really have the same purpose.

I don't even understand why this method is made available in Python, but using ``__mro__`` seems more fitting. (brought up by @mitsuhiko in IRC)